dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java

```diff
@@ -786,6 +786,8 @@ public void addManagedBeanDescriptors(Collection<ManagedBeanDescriptor<?>> manag
      */
     @Override
     public void addEjbDescriptor(EjbDescriptor<?> ejbDescriptor) {
+        Tr.entry(tc, "addEjbDescriptor. Adding EjbDescriptor: " + ejbDescriptor + " with ejbName: " + ejbDescriptor.getEjbName() + " to bda: " + getHumanReadableName());
```

Comment:
What is the EjbDescriptor.toString()?

Reply:
one example of EjdDiscriptor.toString() `WebSphereEjbDescriptor:ejbTimer#ejbTimer.war#EjbSessionBean`

Reply:
I think that in most cases, the ejbName will be included in the ejbDescriptor.toString(). However, a little more trace won't hurt.